build/run

```diff
@@ -120,14 +130,19 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v ${HOME}/.docker/config.json:${BUILDER_HOME}/.docker/config.json \
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
+DOCKER_API=''
+
+if (( $(bc <<< "$DOCKER_SERVER_API < $DOCKER_CLIENT_API") )); then
+    DOCKER_API="-e DOCKER_API_VERSION=${DOCKER_SERVER_API}"
+fi
```

Comment:
Thinking is: at one point we will slow down picking up new CoreOS builds to stabilize => docker in container may fail behind upstream version. If then someone is using a newer version on the host and requests it's higher API version which the container version does not support, things will probably fall apart again. We are fine as long SERVER_API <= CLIENT_API.